velox/docs/functions/spark/misc.rst

```diff
@@ -15,6 +15,17 @@ Miscellaneous Functions
         SELECT at_least_n_non_nulls(2, 0, 1.0, NULL);  -- true
         SELECT at_least_n_non_nulls(2, 0, array(NULL), NULL);  -- true
 
+.. spark:function:: get_struct_field(input, ordinal) -> T
+
+    Returns the value of nested subfield in the ``input`` struct.
```

Comment:
nit: 
Returns the value of nested subfield at position \``ordinal\`` in the input \``struct``.